app/src/components/market/common/outcome_table/index.tsx

```diff
@@ -199,12 +200,16 @@ export const OutcomeTable = (props: Props) => {
         {disabledColumns.includes(OutcomeTableValue.OutcomeProbability) ? null : withWinningOutcome ? (
           <TDStyled textAlign={TableCellsAlign[1]}>
             <BarDiagram outcomeIndex={outcomeIndex} outcomeName={outcomeName} probability={probability} />
-            {showSharesAndPriceChange && <OwnedShares outcomeIndex={outcomeIndex} value="250.55" />}
+            {!shares.isZero() && disabledColumns.includes(OutcomeTableValue.Shares) && (
```

Comment:
Wouldn't this be `!disabledColumns.includes(...)`?

Reply:
Btw, if that's the case, it's probably better to move it to a `showSharesColumn` variable.

Reply:
Oh wait, I get it now. This means that the "You own..." message is only shown when the Shares column isn't. I still think it should be extracted to something like `showOwnedSharesMessage`.

Reply:
Yeah, agreed.